src/cdk/tree/tree.ts

```diff
@@ -297,14 +300,15 @@ export class CdkTree<T> implements AfterContentChecked, CollectionViewer, OnDest
 @Directive({
   selector: 'cdk-tree-node',
   exportAs: 'cdkTreeNode',
-  host: {
-    '[attr.aria-expanded]': 'isExpanded',
-    '[attr.aria-level]': 'level + 1',
-    '[attr.role]': 'role',
-    'class': 'cdk-tree-node',
-  },
 })
-export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
+export class CdkTreeNode<T> implements FocusableOption, OnDestroy, OnInit {
+  /**
+   * The role of the node should always be 'treeitem'.
+   */
+   // TODO: mark as deprecated
+   // tslint:disable:no-host-decorator-in-concrete
```

Comment:
Adding `//` comments between the property and the JsDoc breaks the connection of the description from the property for the generated API docs